srcs/pressure_controller.cpp

```diff
@@ -505,50 +520,35 @@ void PressureController::updateDt(int32_t p_dt) { m_dt = p_dt; }
 
 #if VALVE_TYPE == VT_FAULHABER
 void PressureController::updateOnlyBlower() {
-    int16_t plateauDelta = m_maxPlateauPressureCommand - m_plateauPressure;
+    int16_t peakDelta = m_peakPressure - m_plateauPressure;
 
-    DBG_DO(Serial.println("updatePeakPressure");)
     // If plateau was reached quite early
-    if (m_plateauStartTime < ((m_centiSecPerInhalation * 10u) / 100u)) {
-        DBG_DO(Serial.println("BLOWER -20");)
-
-        // If the peak delta is high, decrease blower's speed a lot
-        if (plateauDelta < -20) {
-            m_blower_increment = -60;
-            DBG_DO(Serial.print("BLOWER -60, peak: ");)
-            DBG_DO(Serial.println(plateauDelta);)
-        } else {
-            m_blower_increment = -20;
-            DBG_DO(Serial.println("BLOWER -20");)
-        }
-    } else if ((m_plateauStartTime >= ((m_centiSecPerInhalation * 10u) / 100u))
-               && (m_plateauStartTime < ((m_centiSecPerInhalation * 20u) / 100u))) {
-        DBG_DO(Serial.println("BLOWER -10");)
-        m_blower_increment = -10;
-    } else if ((m_plateauStartTime > ((m_centiSecPerInhalation * 30u) / 100u))
-               && (m_plateauStartTime <= ((m_centiSecPerInhalation * 40u) / 100u))
-               && abs(plateauDelta) > 10) {
-        DBG_DO(Serial.println("BLOWER +10");)
-        m_blower_increment = +10;
-    } else if (m_plateauStartTime > ((m_centiSecPerInhalation * 40u) / 100u)) {
-        if (plateauDelta > 60) {
-            m_blower_increment = +60;
-            DBG_DO(Serial.print("BLOWER +60, peak: ");)
-            DBG_DO(Serial.print(m_maxPlateauPressureCommand);)
-            DBG_DO(Serial.print("-");)
-            DBG_DO(Serial.println(m_plateauPressure);)
-        } else if (plateauDelta > 40) {
-            m_blower_increment = +40;
-            DBG_DO(Serial.print("BLOWER +40, peak: ");)
-            DBG_DO(Serial.println(plateauDelta);)
-        } else {
-            m_blower_increment = +20;
-            DBG_DO(Serial.println("BLOWER +20");)
-        }
+    // TODO 20 prop to interval
```

Comment:
Can we get rid of this TODO? By implementing missing stuff or because it's no longer needed.

Reply:
Done